style.css

```diff
@@ -0,0 +1,94 @@
+header {
+  display: flex;
+  padding: 10px 10px 10px 10px;
```

Comment:
`padding: 10px` would have the same effect. its shorthand method to apply padding to all directions if its same for each.

Reply:
fixed!